src/routes/(console)/+layout.ts

```diff
@@ -67,3 +61,19 @@ function toPlanMap(plansArray: PlanList | null): Map<Tier, Plan> {
 
     return map;
 }
+
+function makeAsyncCallsIfNecessary() {
+    const { endpoint, project } = sdk.forConsole.client.config;
+
+    if (get(version) === null) {
+        fetch(`${endpoint}/health/version`, {
+            headers: { 'X-Appwrite-Project': project }
+        })
+            .then((res) => res.json().catch(() => null))
+            .then((data) => version.set(data?.version));
+    }
+
+    if (get(consoleVariables) === null) {
+        sdk.forConsole.console.variables().then((vars) => consoleVariables.set(vars));
+    }
```

Comment:
we shouldnt rely on stores in +layout/+page.ts files 👍🏻 its a bit of an antipattern in svelte kit